invenio_app_ils/records/loaders/schemas/changed_by.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""ChangedBy schema for marshmallow loader."""
+
+from flask import has_request_context
+from flask_login import current_user
+from marshmallow import EXCLUDE, Schema, fields, validate
+
+from invenio_app_ils.records.api import IlsRecord
+
+
+class ChangedBySchema(Schema):
```

Comment:
@KonstantinaStoikou I have refactored your code to re-use this in ILL and acquisition. Feel free to add any comment.